scale/job/models.py

```diff
@@ -3660,56 +2622,19 @@ class JobType(models.Model):
     paused = models.DateTimeField(blank=True, null=True)
     last_modified = models.DateTimeField(auto_now=True)
 
-    # TODO: remove with v5 API and/or legacy job types
-    category = models.CharField(db_index=True, blank=True, max_length=50, null=True)
-    author_name = models.CharField(blank=True, max_length=50, null=True)
-    author_url = models.TextField(blank=True, null=True)
-    is_operational = models.BooleanField(default=True)
-    uses_docker = models.NullBooleanField(default=True)
-    docker_privileged = models.NullBooleanField(default=False)
-    docker_params = django.contrib.postgres.fields.JSONField(default=dict, null=True)
-    error_mapping = django.contrib.postgres.fields.JSONField(default=dict, null=True)
+    # deprecated - remove trigger_rule when remove triggers
```

Comment:
weren't triggers already removed?

Reply:
Looks like I removed all references to the field, but never actually removed the field from the job model when triggers were removed. No issues come up removing the field from the model so I'll update the migration to also remove this field.

Reply:
Removed 👍